lib/src/value/color.dart

```diff
@@ -89,6 +89,8 @@ class SassColor extends Value {
   /// indicates a [missing] component.
   ///
   /// [missing]: https://www.w3.org/TR/css-color-4/#missing
+  ///
+  /// @nodoc
```

Comment:
I'm curious why the `@nodoc` was added

Reply:
All of the channel-specific getters are intended to be package-private, I just forgot to add `@nodoc` earlier.